docs/api/server.md

```diff
@@ -40,13 +42,18 @@ function Component() {
 }
 ```
 
-In this code snippet regardless of whether we are server rendering this component or client rendering it, the `logHello` function generates a log on the server console only. How does it work? We use compilation to transform the `server$` function into an RPC call to the server. 
+In this example, regardless of whether we are rendering this component on the server or in the browser, the `logHello` function generates a log on the server console only. How does it work? We use compilation to transform the `server$` function into an RPC call to the server.
```

Comment:
Add the meaning of an RPC call somewhere, maybe as an aside note or a footnote. This will help the less savvy readers grasp the concept of server$